server/config_test.go

```diff
@@ -113,7 +113,8 @@ func TestConfigValidateDefaultSuccessOnExecution(t *testing.T) {
 	defaultConfig.Runtimes["runc"] = oci.RuntimeHandler{RuntimePath: validPath}
 	defaultConfig.Conmon = validPath
 	defaultConfig.NetworkConfig.NetworkDir = validPath
-	defaultConfig.NetworkConfig.PluginDir = []string{validPath}
+	defaultConfig.NetworkConfig.PluginDir = []string{"cni-test"}
```

Comment:
Can we make this a temp directory? `cni-test` looks very generic, so there is a chance we delete local data.

Reply:
Yep sure, I will change it.